aboutmejs.js

```diff
@@ -0,0 +1,86 @@
+ // 
+ const page = document.querySelector('body');
+ page.style.fontFamily='arial';
+
+
+
+
+// 
+document.getElementById('nickname').innerText = 'Munira AlShuhail';
+document.getElementById('favorite').innerText = 'None.. I hate animals!';
+document.getElementById('hometown').innerText = 'Saudi Arabia, AlAhsa';
+
+
+//
+
+document.getElementById('nickname');
+nickname.style.color='rebeccapurple';
+document.getElementById('favorite');
+favorite.style.color='rebeccapurple';
+document.getElementById('hometown');
+hometown.style.color='rebeccapurple';
+
+//
+const listItem = document.querySelector('ul');
+listItem.style.color='rebeccapurple';
+
+
+
+
+// var items = document.getElementsByTagName('li');
+// for (var i = 0; i>= items.length; i++) {
+//     items[i].className = 'listitem';
+//  }
```

Comment:
You were on the right track here, the only thing that was stopping your code from working is your loop condition. Your `i` starts at 0 and increments in every iteration, so your condition needed to be `i< items.length` instead of `>=`